src/store/allIncidentsSlice/thunks/util.js

```diff
@@ -17,3 +19,47 @@ export function selectListByStatus(status, state) {
       return null;
   }
 }
+
+/**
+ * takes a 'dict' with numbered keys and returns an array
+ * This is used because some of the DS endpoints send back a dictionary instead of a list
+ *
+ * @param {any} dict
+ * @return {any[]}
+ */
+function dictionaryToArray(dict) {
+  if (dict == null) {
+    return [];
+  }
+
+  const array = [];
+
+  for (const key in dict) {
```

Comment:
Object.Keys()

Object.Entries()

might be something to consider here as well.